src/index.tsx

```diff
@@ -34,9 +39,44 @@ function loadInitialData() {
   store.dispatch(readEndpoint('/annotations?url=' + window.location.href));
 }
 
+export function loadDataFromChromeStorage() {
```

Comment:
All other actions taking place here are just one-liners and this function take most of the file, is it that important? How about moving it to less prominent place?

Reply:
👍, I'd say it should be moved to some utils/actions/ file